modules/log4j2/src/main/java/org/apache/ignite/logger/log4j2/Log4J2Logger.java

```diff
@@ -17,11 +17,11 @@
 package org.apache.ignite.logger.log4j2;
 
 import java.io.File;
-import java.lang.reflect.Field;
 import java.net.URL;
 import java.nio.charset.Charset;
 import java.util.Map;
 import java.util.UUID;
+
```

Comment:
According to the code style guideline, there should be no empty lines between the imports.

Reply:
Thanks, fixed with latest commit.